### PR TITLE
Adding tracking of human feedback

### PR DESCRIPTION
**Pull request by zmhassan**
## Description
Adding tracking ability for human in the loop feedback. As we want to track how many false anomalies are reported vs true anomalies.

### CODE REVIEW

**Comment by zmhassan** [comment]
In prometheus metrics will look like this:
``` aiops_human_feedback_count_total{anomaly_status="False",customer_id="#1234567"} 1.0```

File: anomaly_detector/fact_store/app.py [hunk 1]
```diff
@@ -74,17 +81,5 @@ def feedback():
     return ""
 
 
-@app.route("/api/anomaly_event", methods=["POST"])
```
Comment (durandom) [inline anomaly_detector/fact_store/app.py]: why do you remove this in this PR? it's not related to the counter.

Comment (zmhassan) [reply anomaly_detector/fact_store/app.py]: We are no longer recording events as its data duplication. We have single table which records just the feedback and the log line which is not an anomaly.

**Comment by zmhassan** [comment]
> I'm ok with it. But wouldnt it be easier to track in LAD Core, when you download the feedback data. This way you don't have to add another target to prometheus (the fact-store)

We can do that. However my thought was that this is more simple to update in future and also I don't want to make LAD Core dependent on Fact Store. If we install fact_store we get to track metrics

**Pull request merged**